rundeckapp/build.gradle

```diff
@@ -102,8 +102,8 @@ dependencies {
     compile "org.springframework.boot:spring-boot-autoconfigure"
 
     compile "org.grails:grails-core"
-    //compile "org.springframework.boot:spring-boot-starter-actuator"
-    compile ("org.springframework.boot:spring-boot-starter-jetty"){
+    compile "org.springframework.boot:spring-boot-starter-actuator"
```

Comment:
is the actuator necessary?

Reply:
The actuator was added to expose the health endpoint and also to add a refresh endpoint. After further research I'm not sure we really need this. I was using it as a quick way to expose the /health check endpoint, but we can do that ourselves without the other features that actuator also adds.